pkg/controllermanager/controller/factory.go

```diff
@@ -20,11 +20,12 @@ import (
 
 	gardencore "github.com/gardener/gardener/pkg/apis/core"
 	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
-	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	gardencorev1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
```

Comment:
Why was this renamed? :) I think `v1beta1constants` is usually used.

Reply:
I ran my formatter to ensure consistent imports, but I was already wary in this and the changed dot-import for the tests. Guess I just revert the last commit.